org.adempiere.base/src/org/eevolution/model/MDDOrder.java

```diff
@@ -1274,9 +1274,10 @@ public int getC_Currency_ID() {
 	public boolean isComplete()
 	{
 		String ds = getDocStatus();
-		return DOCSTATUS_Completed.equals(ds) 
+		return DOCSTATUS_Completed.equals(ds)
 			|| DOCSTATUS_Closed.equals(ds)
-			|| DOCSTATUS_Reversed.equals(ds);
+			|| DOCSTATUS_Reversed.equals(ds)
+			|| DOCSTATUS_Voided.equals(ds);

```

Comment:
this is now inconsistent with isComplete() in other model class

Reply:
solved